### PR TITLE
Update evohome, honeywell markdown with latest changes

### PR DESCRIPTION
**Pull request by zxdavb**
**Description:**
Add zones to evohome platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18428

Note this is for branch **next**

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by zxdavb** [comment]
@frenck Would you mind kindly reviewing this? - it is the only thing holding up my PR (https://github.com/home-assistant/home-assistant/pull/18428), which has just been merged into dev.

Note that it is for the `next` release of HA as it refers to changes in the `evohome` platform that wont exist until then.

File: source/_components/climate.evohome.markdown
Comment (MartinHjelmare) [inline source/_components/climate.evohome.markdown]: This page describes the evohome climate platform. The evohome component is the last modified page in the PR.

```suggestion
The `evohome` climate platform integrates your _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) system into Home Assistant. It has been tested with the Honeywell evohome multi-zone CH/DHW system, but other systems may also work.
```

File: source/_components/climate.evohome.markdown
Comment (MartinHjelmare) [inline source/_components/climate.evohome.markdown]:
```suggestion
It is distinct from - but related to - the [`honeywell`](/components/climate.honeywell/) climate platform, which also allows (limited) integration with Honeywell Connected thermostats; these two components may well be usable side-by-side, but YMMV.
```

File: source/_components/climate.honeywell.markdown
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
Although both regions remain supported here, the `evohome-client`-based systems have a new [component](/components/evohome/) available that may suit your needs better.
```

File: source/_components/climate.honeywell.markdown
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
To set up this climate platform, add the following information to your `configuration.yaml` file:
```

File: source/_components/climate.honeywell.markdown
Comment (MartinHjelmare) [inline source/_components/climate.honeywell.markdown]:
```suggestion
Scan interval is expressed in seconds. Omitting or mis-configuring `scan_interval` may result in too-frequent polling and cause you to be rate-limited by Honeywell.
```

File: source/_components/evohome.markdown
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
The `evohome` component is the main component to set up and integrate all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, primarily the Honeywell evohome multi-zone CH/DHW system.  It does not support the home security functionality of TCC.
```

File: source/_components/evohome.markdown
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
It uses v2 of the [evohome-client](https://github.com/watchforstock/evohome-client) client library and so will let you control (only) EU/international systems. It _does not_ leverage the [somecomfort](https://github.com/kk7ds/somecomfort) client library as used by US-based systems; for those, you may find what you need at the [honeywell climate platform](/components/climate.honeywell/).
```

File: source/_components/evohome.markdown
Comment (MartinHjelmare) [inline source/_components/evohome.markdown]:
```suggestion
Currently, only Controllers and Heating zones are supported; support for DHW controllers will be added at a later time. More information is available on the [evohome climate platform](/components/climate.evohome/) page.
```

**Comment by zxdavb** [comment]
Clearly, I still haven't got my head around the nomenclature, and the difference between a platform & a component!

Thanks for your help.

**Review by MartinHjelmare — APPROVED** [review]
Can be merged when build passes.

**Pull request merged**
